app/controllers/provider_foreman_controller.rb

```diff
@@ -380,6 +395,14 @@ def cs_filter_tree_rec
     end
   end
 
+  def configuration_scripts_tree_rec
+    nodes = x_node.split('-')
```

Comment:
@lgalis This is somewhat unrelated to this PR, but since I see it here... What is `x_node`?
I've seen it all over the UI and it always seems to be split.  Although sometimes it's split on `-` and other times `-` and `_`.  That leads me to believe that there are 3 parts, and based on the variables I think it's `#{node}_#{type}-#{id}`.  Is it that simple and consistent?  In this case, it looks like there are only two parts.